mmdet3d/datasets/kitti_dataset.py

```diff
@@ -32,6 +32,14 @@ class KittiDataset(Det3DDataset):
             - 'LiDAR': Box in LiDAR coordinates.
             - 'Depth': Box in depth coordinates, usually for indoor dataset.
             - 'Camera': Box in camera coordinates.
+        load_type (str, optional): Type of loading mode.
```

Comment:
It seems the `load_type` design conflicts with the unified `task` design.

Reply:
Whether `lidar_det`, `multi-view_det` and `mono_det` can match `frame_based`, `mv_image_based`, `for_image_based`?

Reply:
Cannot. lidar_det corresponds to frame_based and mono_det corresponds to fov_image_based exactly, while multi-view_det may correspond to frame_based/mv_image_based.